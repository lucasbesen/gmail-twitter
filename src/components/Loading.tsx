import React from 'react';
import styled from 'styled-components';
import CircularProgress from '@material-ui/core/CircularProgress';

const Wrapper = styled.div`
  display: flex;
  flex: 1;
  justify-content: center;
  align-items: center;
  align-self: center;
  height: 50%;
`;

const Loading: React.FC = () => (
  <Wrapper>
    <CircularProgress color="secondary" />
  </Wrapper>
);

export default Loading;
