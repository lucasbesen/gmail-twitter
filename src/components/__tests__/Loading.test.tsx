import React from 'react';
import renderer from 'react-test-renderer';

import Loading from '../Loading';

it('should render Loading without error', () => {
  const component = renderer.create(<Loading />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
