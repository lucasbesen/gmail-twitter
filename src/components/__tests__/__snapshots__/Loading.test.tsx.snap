// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Loading without error 1`] = `
<div
  className="sc-bdVaJa jFWyCK"
>
  <div
    className="MuiCircularProgress-root MuiCircularProgress-colorSecondary MuiCircularProgress-indeterminate"
    role="progressbar"
    style={
      Object {
        "height": 40,
        "width": 40,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svg"
      viewBox="22 22 44 44"
    >
      <circle
        className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
        cx={44}
        cy={44}
        fill="none"
        r={20.2}
        strokeWidth={3.6}
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
